### PR TITLE
Model ingredients

### PR DESCRIPTION
**Pull request by beckystar4**
Made Ingredient model and migrated it with the database

Added a button on the homepage to get to ingredient list.

Made page that lists all the names of ingredients in the database

Validation is only on the name for right now, but plan to add more

### CODE REVIEW

File: db/migrate/20230224185802_create_ingredients.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    create_table :ingredients do |t|
+      t.string :name
+      t.timestamps
+      t.string :type
+      t.boolean :is_alcoholic
+      t.date :vintage
+      t.string :size
+      t.float :abv
+      t.string :purchase_url
+      t.blob :image
```
Comment (JMGbaum) [inline db/migrate/20230224185802_create_ingredients.rb]:
```suggestion
```
We can remove this line. Images are actually done in the model file. They do not need to be set up in a migration.

File: Gemfile.lock [hunk 2]
```diff
@@ -73,10 +73,9 @@ GEM
     bindex (0.8.1)
     bootsnap (1.15.0)
       msgpack (~> 1.2)
-    bootstrap (5.2.3)
```
Comment (JMGbaum) [inline Gemfile.lock]: The Gemfile should not be changing.

File: db/migrate/20230224185802_create_ingredients.rb (hunk 1 above)
Comment (JMGbaum) [reply db/migrate/20230224185802_create_ingredients.rb]: I'm setting up what we need for images in my PR, so I can help with this later.

**Review by JMGbaum — APPROVED** [review]

**Review by EliCoop — APPROVED** [review]

**Comment by JMGbaum** [comment]
[Create Ingredient Model](https://trello.com/c/ncF7sQ5z/19-create-ingredient-model)

**Comment by JMGbaum** [comment]
[Create Ingredient Page](https://trello.com/c/NM6keSm9/40-create-ingredient-page)

**Pull request merged**